docs/en/sql-reference/aggregate-functions/reference/grouparraysorted.md

```diff
@@ -0,0 +1,35 @@
+ ---
+ toc_priority: 108
```

Comment:
top_priority defines the position of the entry in the sidebar. Please double-check it will be next to the existing groupArray* agg functions